.travis.yml

```diff
@@ -12,6 +12,19 @@ env:
   - PUPPET_VERSION=3.5 FUTURE_PARSER=yes TRUSTED_NODE_DATA=yes
   - PUPPET_VERSION=4.0
 matrix:
+  include:
+    - rvm: default
+      sudo: required
+      services: docker
```

Comment:
Can you use docker on travis inside a container? We use sudo: false and IIRC that was not (yet) supported.